SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -113,63 +128,68 @@ etc...
 
     > These lines ensure that the SpatialOS Unreal GDK can override Unreal's network replication.
     >
-    > If you ever need to run with Unreal networking for workflow or validation reasons, untick `Spatial Networking` in the `Play` drop down menu in the Unreal editor. This flag is maintained in your Unreal config files under `/Script/EngineSettings.GeneralProjectSettings` and is also valid for non-editor builds.
-    >
-    > You can easily switch between networking modes using this checkbox. As the SpatialOS Unreal GDK is in pre-alpha, switching back to Unreal's networking may be a viable way to work around some issues you may encounter, and may speed up your iteration. However, in Unreal networking mode you lose access to key multi-server features of the GDK, which may lead to erratic behavior.
+    > If you ever need to run with default Unreal networking for workflow or validation reasons, un-tick `Spatial Networking` in the `Play` drop down menu in the Unreal editor. This flag is maintained in your Unreal config file `<GameRoot>/Config/DefaultGame.ini` under `/Script/EngineSettings.GeneralProjectSettings` and is also valid for non-editor builds.
```

Comment:
In the list item - "In <GameRoot>\Config, open DefaultEngine.ini and add:"
(I can't get to it because it's not a diff but the edits further down mean we need to edit this bit....)
Make:
" 1. In <GameRoot>\Config, open DefaultEngine.ini and add:"
->
" 1. Set up SpatialOS networking. <br/>
In <GameRoot>\Config, open DefaultEngine.ini and add:"

----
This makes the step below make sense.